scripts/CEE/registry-s3-bucket-size/script.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Compute the size of the bucket backing the in-cluster container registry
+# Only AWS clusters with S3 buckets backing the registry are supported
+
+set -e
+
+get_s3_bucket_name() {
+  oc get configs.imageregistry.operator.openshift.io/cluster -ojsonpath='{.spec.storage.s3.bucket}'
+}
+
+get_s3_access_accesskey() {
+  local b64enckey=
+  b64enckey=$(oc -n openshift-image-registry get secret image-registry-private-configuration -o jsonpath='{.data.credentials}')
+  echo "${b64enckey}" | base64 --decode |  grep aws_access_key_id | cut -d= -f2- | tr -d "[:blank:]"
+}
+
+get_s3_access_secretkey() {
+  local b64enckey=
+  b64enckey=$(oc -n openshift-image-registry get secret image-registry-private-configuration -o jsonpath='{.data.credentials}')
+  echo "${b64enckey}" | base64 --decode |  grep aws_secret_access_key | cut -d= -f2- | tr -d "[:blank:]"
+}
+
+bucket_size() {
+  local bucket="${1}" accesskey="${2}" secretkey="${3}"
+  AWS_ACCESS_KEY_ID="${accesskey}" AWS_SECRET_ACCESS_KEY="${secretkey}" aws s3api list-objects --bucket "${bucket}" --output json --query "sum(Contents[].Size)"
```

Comment:
is that command region specific? if so, should we pass the  `AWS_DEFAULT_REGION` aswell

Reply:
I dont think `s3api` command is region specific, eg: https://docs.aws.amazon.com/cli/latest/reference/s3api/list-buckets.html#examples

I also the script on two clusters that were installed in two different regions (`ap-southeast-2` and `us-west-2` )